src/open_inwoner/accounts/templates/registration/login.html

```diff
@@ -1,5 +1,5 @@
 {% extends 'master.html' %}
-{% load i18n static logo_tags grid_tags card_tags form_tags link_tags button_tags solo_tags cms_tags %}
+{% load i18n static ssd_tags logo_tags grid_tags card_tags form_tags link_tags button_tags solo_tags cms_tags %}
```

Comment:
can we move the markdown filter from ssd_tags into the utils tags if we're using it in other places too?